assets/multiqc_config.yaml

```diff
@@ -1,17 +1,11 @@
 report_comment: >
     This report has been generated by the <a href="https://github.com/nf-core/methylseq" target="_blank">nf-core/methylseq</a>
     analysis pipeline. For information about how to interpret these results, please see the
-    <a href="https://github.com/nf-core/methylseq/blob/master/docs/README.md" target="_blank">documentation</a>.
+    <a href="https://github.com/nf-core/methylseq" target="_blank">documentation</a>.
 report_section_order:
     software_versions:
         order: -1000
     nf-core-methylseq-summary:
         order: -1001
-table_columns_visible:
```

Comment:
We don't have to delete this - the test just checks that this file _starts_ the same as the template, so we can move this custom stuff to the end and it should pass

Reply:
oops, added it back

Reply:
It's a bit too easy to do with `nf-core lint --fix`! Not sure if there's a better way.. 🤔